client/js/place.js

```diff
@@ -456,6 +457,34 @@ var place = {
         this.didSetHash = true;
     },
 
+    contextMenu: function(event) {
+        event.preventDefault();
+        let zoom = place._getZoomMultiplier();
+        let x = Math.round((event.pageX - $(place.cameraController).offset().left) / zoom)
+        let y = Math.round((event.pageY - $(place.cameraController).offset().top) / zoom)
+        return $.get(`/api/pixel?x=${x}&y=${y}`).done(data => {
+            if(data.pixel == null) data.pixel = { // No pixel was clicked... For simplicity we just change the response
+                editor: {
+                    username: "Nobody!",
+                    statistics: {
+                        totalPlaces: 0
+                    }
+                },
+                modified: 0
+            };
+            if(!data.success) return failToPost(data.error);
+            $("#coordinate-modal > .modal-dialog > .modal-content > .modal-body > b > #owner").text(()=>{return data.pixel.editor.username});
```

Comment:
I don't think we should use a modal, I've been coding a popover for this, but haven't had the chance to put the finishing touches on this